apps/test-suite/tests/Image.js

```diff
@@ -182,5 +197,15 @@ export async function test(t, { setPortalChild, cleanupPortal }) {
         }
       });
     });
+
+    t.describe('getBlurhashAsync', async () => {
+      t.it('returns a correct blurhash for url', async () => {
+        const result = await Image.getBlurhashAsync(REMOTE_KNOWN_SOURCE.uri, [4, 3]);
+        t.expect(result).toBe(REMOTE_KNOWN_SOURCE.blurhash);
+      });
+      t.it('rejects on a missing url', async () => {
+        await throws(Image.getBlurhashAsync(NON_EXISTENT_SOURCE.uri, [4, 3]));
```

Comment:
```suggestion
        await expect(async () => {
          await Image.getBlurhashAsync(NON_EXISTENT_SOURCE.uri, [4, 3]);
        }).rejects.toThrow();
```

Reply:
<img width="350" alt="image" src="https://github.com/expo/expo/assets/5597580/13a3b8f9-0e1f-4089-8ad2-da1c6fc6146d">
Are you sure that works? It was my first try as well, but it seems out `t.expect` doesn't contain the `.rejects` field.

Reply:
Yeah, that should work in newer versions of Jest 🤔 It seems odd to have to use these helpers like `throws` 😅